corehq/apps/es/case_search.py

```diff
@@ -68,6 +72,24 @@ def case_property_query(self, case_property_name, value, clause=queries.MUST, fu
         else:
             return self.add_query(exact_case_property_text_query(case_property_name, value), clause)
 
+    def related_case_property_query(self, domain, path, value):
+        """
+        Search for property in a related case.
+
+        Supports a single-level path, e.g., parent/property_name
+
+        Does not support fuzziness.
+        """
+        try:
+            (identifier, prop) = path.split("/")
```

Comment:
I know you are only calling this if there's a `/` in `path` but I think it would also be good to check here as a precondition
```

Reply:
It'll throw the `ValueError` because there'll be one value when it's expecting two.